pom.xml

```diff
@@ -136,8 +137,7 @@
       <version.micrometer>1.6.1</version.micrometer>
       <hamcrest.version>2.1</hamcrest.version>
       <surefire.version>2.22.2</surefire.version>
-      <version.jaxb.api>2.3.1</version.jaxb.api>
-      <version.jaxb.runtime>2.3.3</version.jaxb.runtime>
+      <jaxb.runtime.version>2.3.3</jaxb.runtime.version>
```

Comment:
Why did you rename *version.jaxb.runtime* into *jaxb.runtime.version* ?

Reply:
it was just to use same convection of other version properties but I don't need it in this commit so I have reverted it